### DISCUSSION TITLE
Content sort filter framework

### DISCUSSION THREAD

**Discussion by Angelk90** [TeamNewPipe/NewPipe#10554] [Questions]
Hi, regarding the following pull request https://github.com/TeamNewPipe/NewPipe/pull/8837 created by @evermind-zz.

I would like to understand why it was never accepted, what reasons blocked its approval.

I have to have two versions of the same app, because one doesn't allow you to filter videos and the other because it isn't updated to the latest version.

**Comment by opusforlife2** [comment]
You were answered here: https://github.com/TeamNewPipe/NewPipe/pull/8837#issuecomment-1648559297
